core/schemas/observable.py

```diff
@@ -24,6 +28,7 @@ class ObservableType(str, Enum): ...
 
 ObservableTypes = ()
 TYPE_MAPPING = {}
+FileLikeObject = Union[str, os.PathLike, IO, tempfile.SpooledTemporaryFile]
```

Comment:
Let's settle on one notation (at least throughout the file); either use `Union` or use ` | ` (I have a slight preference for the latter)

Reply:
Fixed in 1c5cc8f0abd23d6e1dab6589460acad05faa5452